mysystehis/main.tex

```diff
@@ -0,0 +1,1155 @@
+ \section{\Large Objective} 
+ 	In this project, since it's about building an RPG game, which is also a type of virtual world, I'll only focus on how to make the player move around the world, is able to interact with other objects around him. In this game, there may be some enemies that get in the way, the player should eliminate them which hurt the player. 
+ \section{\Large Aim} 
+ 	The aim is not necessarily on destroying the enemies, but focus on completing 
+ 	the quest that villagers in the village gave the player. When finishing a quest, the player should return back to the village, talk to the villagers that you're finished. Beside that, the game should also have an online chat tab , which you can chat with other players who are connecting to the same network while playing game, they may help you finishing the quest faster. 
+
+ 
+ 
+
+ \titleformat{\paragraph}
+ {\normalfont\normalsize\bfseries}{\theparagraph}{1em}{}
+ \titlespacing*{\paragraph}
+ {0pt}{1.25ex plus 1ex minus .2ex}{1.5ex plus .2ex}
+ 
+ \vspace{0.35 cm}
+  \section{\Large Programs, Materials and Methods}
+  	\subsection{External programs used}
+  		\subsubsection{3DS Max}
+ 		 3DS Max, a program developed by Autodesk, previously called 3D Studio Max, is a 3D computer graphics program for making 3D animations, models , images...
+ 		 This program was used in this project in order to edit UV mapping of some models, correct the position of the texture. 
+ 		 \subsubsection{Photoshop}
+ 		 Photoshop is a photo editing and graphic design software. It is developed by Adobe Systems for MacOS and Windows. It's not like other graphic design or photo editing softwares, it can create normal map, height map, occulusion map from a single texture. With this, this software will help me with the technique bump mapping, so the game's texture will look more realistic. 
+ 		 \subsubsection{Visual Studio}
+ 		 Visual studio is an IDE (integrated development environment ) from Microsoft. It's used to develop computer programs, as well as websites, supports many programming languages such as : C++, C\#, JavaScript, etc... 
+ 		 This is the main software of this project, because i'm using this one to write scripts in order to make the game works.
+ 		 When installing unity 5, normally it's shipped with visual studio community which is free for everyone. Previously, it's shipped with the monodevelop, but now the monodevelop is discontinued, no longer support unity, replaced with visual studio community as it's more powerful than monodevelop, it support MacOS and windows, gives you a cloud storage for saving like one drive(when you project goes wrong), which monodevelop doesn't have.  
```

Comment:
em dung noi la Visual Studio gives you cloud storage for saving like one drive, boi vi ko ai dung cloud storage de luu source code ca. Cong cu chinh de luu source code phai la 1 he thong SCM (source control management) vi du nhu dung git cung voi github hoac bitbucket.